app/src/main/java/ch/epfl/qedit/model/Server.java

```diff
@@ -0,0 +1,92 @@
+package ch.epfl.qedit.model;
+
+import androidx.annotation.NonNull;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class Server {
```

Comment:
I am wondering if this class should really be called `Server`, that's maybe to generic. And it probably shouldn't be part of the `model` package, but rather in a `backend` package, or the like.

Reply:
I used that name to hide firebase because it really is just a wrapper, an adapter for it.

Reply:
Ok, the name is fine, I agree. But I think we need to move it to another package, becaues it really doesn't belong to the model part of MVVM.